pkg/skaffold/build/local/local.go

```diff
@@ -112,3 +113,11 @@ func (b *Builder) getImageIDForTag(ctx context.Context, tag string) (string, err
 	}
 	return insp.ID, nil
 }
+
+func ToDockerOpts(opts *build.ImageOptions) *docker.BuildOptions {
```

Comment:
This should be done in the docker builder itself.

Reply:
Pushing `ImageOptions` into any of the actual builder implementations causes an import cycle (you had predicted this!). So I've defined `ImageOptions` as the generic builder-agnostic options object. The different builders define their own options object in their respective adapter package.